pkg/storage/gcs/store.go

```diff
@@ -58,7 +58,7 @@ func (g *gcs) Has(ctx context.Context, objectName string) (bool, error) {
 	}
```

Comment:
The method should be updated to use the client that is part of the [gcs instance](https://github.com/oneconcern/datamon/pull/233/files#diff-aaf1bc657e525c33e0d1422846c07b12R97)